projects/project-player-library/src/lib/pages/task-details-page/task-details-page.component.ts

```diff
@@ -154,10 +156,25 @@ export class TaskDetailsPageComponent implements OnInit {
         if(data.isChecked && data.upload){
           this.routingService.navigate(`/add-files/${this.projectDetails._id}`,{taskId:this.taskId})
         }else{
-          this.toasterService.showToast('ACCEPT_POLICY_ERROR_MSG')
+          this.toasterService.showToast('ACCEPT_POLICY_ERROR_MSG',"danger")
         }
       }
     })
   }
 
+  onDateChange(newDate: Date) {
+    let localDateString = this.formatDateToLocal(newDate);
+    this.task.endDate = localDateString;
+    this.updateDataInDb();
+  }
+
+  formatDateToLocal(date: Date): string {
+    let offset = date.getTimezoneOffset() * 60000;
+    let localISOTime = new Date(date.getTime() - offset).toISOString().slice(0, -1);
+    return localISOTime;
+  }
+
+  onLearningResources(id:any,fromHome:boolean){
```

Comment:
Change the key from fromHome to relevant name

Reply:
done